hazelcast/src/main/java/com/hazelcast/durableexecutor/impl/operations/PutResultOperation.java

```diff
@@ -58,6 +65,17 @@ public WaitNotifyKey getNotifiedKey() {
     }
 
     @Override
+    public boolean shouldBackup() {
+        Version version = getNodeEngine().getClusterService().getClusterVersion();
+        return version.isGreaterOrEqual(V3_9);
+    }
+
+    @Override
+    public Operation getBackupOperation() {
+        return new PutResultBackupOperation(name, sequence, result);
```

Comment:
In relation to #10010 , shouldn't we send either a `PutResult` or `PutResultBackup` operation depending on cluster version?